src/main/java/domain/WinningLotto.java

```diff
@@ -0,0 +1,39 @@
+package domain;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class WinningLotto {
+    private static final int WINNING_LOTTO_NUMBER = 6;
+
+    private final Set<Number> winningNumbers;
+    private final Number bonusNumber;
+
+    public WinningLotto(Set<Number> winningNumbers, Number bonusNumber) {
+        validate(winningNumbers);
+        this.winningNumbers = winningNumbers;
+        checkExist(bonusNumber);
+        this.bonusNumber = bonusNumber;
```

Comment:
개인적으로 유효성 검사는 모든 필드에 대해서 우선적으로 실행하는것을 선호합니다 !
```suggestion
        validate(winningNumbers);
        checkExist(winningNumbers, bonusNumber);
        this.winningNumbers = winningNumbers;
        this.bonusNumber = bonusNumber;
```